google/cloud/sql/connector/connector.py

```diff
@@ -182,22 +183,29 @@ async def connect_async(
             icm.force_refresh()
             raise (e)
 
-    async def _close(self) -> None:
-        """Helper function to close InstanceConnectionManagers' tasks."""
-        await asyncio.gather(*[icm.close() for icm in self._instances.values()])
+    def __enter__(self) -> Any:
```

Comment:
quick afterthought: should we have an `__aenter__` and `__aexit__` for the async interface?

Reply:
I was going to implement these when we add the first async driver, because as of right now everything goes through the sync `connect`.

Reply:
Ok, that sounds reasonable.